Project.toml

```
@@ -59,4 +59,4 @@ StatsBase = "0.32, 0.33"
 StatsFuns = "0.8, 0.9"
 Tracker = "0.2.3"
 ZygoteRules = "0.2"
-julia = "1.3, 1.4, 1.5, 1.6"
+julia = "1.6, 1.7"
```

This is the same as
```suggestion
julia = "1.6"
```
i.e., Julia 1.6 and all newer versions are supported. (The current compat entry seems weird, I guess it was supposed to bound it to < 1.7 but is incorrect.)